app.py

```diff
@@ -0,0 +1,95 @@
+from flask import Flask, request, send_file
```

Comment:
Please fix coding convention 2 spaces or 4 spaces?

Reply:
if you use vs code, command+k, f will help you out

Reply:
Thank you for your advice!